mappings/net/minecraft/client/gui/widget/EntryListWidget.mapping

```diff
@@ -89,7 +89,8 @@ CLASS net/minecraft/class_350 net/minecraft/client/gui/widget/EntryListWidget
 		ARG 1 index
 	METHOD method_25333 setLeftPos (I)V
 		ARG 1 left
-	METHOD method_25334 getSelected ()Lnet/minecraft/class_350$class_351;
+	METHOD method_25334 getSelectedOrNull ()Lnet/minecraft/class_350$class_351;
+		COMMENT Returns the selected entry of this entry list, or {@code null} if there is not.
```

Comment:
```suggestion
		COMMENT {@return the selected entry of this entry list, or {@code null} if there is not.}
```

Reply:
Just fyi, the default behavior of inline return tag is it adds an extra comma, so in the `{@return}` you just put what you would put in a regular block `@return` tag.